test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -32,6 +37,27 @@ public MsbuildIntegrationTestFixture()
 
             var sdkPaths = Directory.GetDirectories(Path.Combine(_cliDirectory, "sdk"));
 
+            //Temporary patching process for System.Security.Cryptography.Pkcs.dll and deps.json files
+            //Will be removed when shipping
+            var patchDir = sdkPaths.Where(path => path.Split(Path.DirectorySeparatorChar).Last().StartsWith("5")).First();
+            TempPatching(patchDir);
+
+            string packagePath = null;
+            if (RuntimeEnvironmentHelper.IsWindows)
+            {
+                packagePath = Path.Combine(Environment.GetEnvironmentVariable("USERPROFILE"), ".nuget", "packages");
+            }
+            else
+            {
+                packagePath = Path.Combine(Environment.GetEnvironmentVariable("HOME"), ".local", "share", "NuGet", "v3-cache");
+            }
+
+            var pathCopyFrom = Path.Combine(packagePath, "system.security.cryptography.pkcs", "5.0.0-alpha1.19473.1", "lib", "netstandard2.1");
+            var pathCopyTo = patchDir;
+            var dlls = new string[1] { "System.Security.Cryptography.Pkcs.dll" };
+            TempCopyNewlyAddedDlls(dlls, pathCopyFrom, pathCopyTo);
+            //end of Temporary patching
+
```

Comment:
How about extracting this patching code into a new method:
```C#
private static void ApplyTemporaryPatch(...)
```

Then you can remove the comments.